packages/eslint-plugin/docs/rules/await-thenable.md

```diff
@@ -0,0 +1,33 @@
+# Disallows awaiting a value that is not a Promise (await-thenable)
+
+This rule disallows awaiting a value that is not a "Thenable" (commonly, a Promise).
```

Comment:
I hope a description of "Thenable". For example, "objects which have `then` method" or something like.